workflow/02_pre_processing.Rmd

```diff
@@ -391,28 +380,26 @@ left_join(hector_temp_slice, run_fail_summary_tgav_ssp, by = c("run_number", "ss
 
 ``` {r filter-temp}
 # Note that the tracking output needs to be filtered as well
-trk_output_filtered_1 <- run_fail_summary_h %>%
-  filter(fraction_fail < MAX_FAILFRAC_HISTORICAL) %>% 
-  left_join(trk_output, by = "run_number") %>% 
-  select(-fraction_fail)
+trk_output_filtered_1 <- run_fail_summary_new %>%
+  filter(fail) %>% 
+  select(-fraction_out, -fail) %>% 
+  left_join(trk_output, by = "run_number")
 
 # Remove large data and force clear memory
 rm(trk_output)
-gc()
 
-trk_output_filtered_2 <- run_fail_summary_tgav %>% 
-  left_join(trk_output_filtered_1, by = c("ssp", "run_number")) %>% 
-  select(-fraction_fail)
+trk_output_filtered_2 <- tgav_h_hist %>% 
+  select(-fraction_out, -fail) %>% 
+  left_join(trk_output_filtered_1, by = "run_number")
```

Comment:
@leeyap Note the ordering change here. Before, we joined and then dropped unneeded columns...but if they're not needed, and we're dealing with big datasets, it's better to drop them _before_ merging.

In other words, let's not make `left_join` do the work (in time and memory use) of creating them if they're just going to be removed.